app/containers/WriteReviewPage/index.js

```diff
@@ -38,23 +41,57 @@ export class WriteReviewPage extends React.PureComponent { // eslint-disable-lin
 
   componentWillReceiveProps(nextProps) {
     // Reset trip rating if selected location changes
-    if(nextProps.selectedLocationId !== this.props.selectedLocationId) {
+    if (nextProps.selectedLocationId !== this.props.selectedLocationId) {
       this.props.handleTripRatingChange(null);
     }
   }
 
   /***** Custom Methods *****/
 
+  // Custom validation rules, not handled by form classes
+  formSubmissionIsValid = () => {
+    return this.props.selectedLocationId && !(this.props.isGuided && !this.props.selectedGuideId);
+  };
+
+  validateForm = (form) => {
+    let valid = true
```

Comment:
Is it OK to have the default value be `true`? I would think it safer to default to `false` and make it `true` only after the proper checks so that a mistake or bug doesn't slip through as valid.

Reply:
I think it's fine here, every block in the function sets it to false. For testing this (automated or manual) we'll need to hit every block anyway

And, the server will double-validate